tests/models/test_state.py

```diff
@@ -75,6 +74,47 @@ def setUp(self):
         )
 
         Score.save_all([self.score1, self.score2, self.score3, self.score4])
+
+        self.innovative_policy_idea1 = InnovativePolicyIdea(
+            subcategory_id=subcategory.id,
+            state=self.state.code,
+        )
+        self.honorable_mention1 = HonorableMention(
+            subcategory_id=subcategory.id,
+            state=self.state.code,
+        )
+        # self.honorable_mention2 = HonorableMention(
+        #     subcategory_id=subcategory2.id,
+        #     state=self.state.code,
+        # )
+        # self.honorable_mention3 = HonorableMention(
+        #     subcategory_id=subcategory2.id,
+        #     state=self.state.code,
+        # )
+        # honorable_mention2 was created more recently than honorable_mention3
+        # self.honorable_mention3.created_at = datetime.utcnow() - timedelta(5)
+        # self.honorable_mention4 = HonorableMention(
+        #     subcategory_id=subcategory.id,
+        #     state=other_state.code,
+        # )
+        # self.innovative_policy_idea2 = InnovativePolicyIdea(
+        #     subcategory_id=subcategory.id,
+        #     state=self.state.code,
+        # )
+        # self.innovative_policy_idea2.deactivate()
```

Comment:
Do we wanna keep these test cases in?

Reply:
Yes apologies! I made all these changes and realized I needed a basic link type, and thus the `ResourceLink` was born, so I had to go back and fix that first. I'm going to update this right now :)